src/Tasks/Microsoft.NET.Build.Tasks/NuGetPackageResolver.cs

```diff
@@ -31,6 +31,19 @@ public string GetPackageDirectory(string packageId, NuGetVersion version)
             return _packagePathResolver.GetPackageDirectory(packageId, version);
         }
 
+        public string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot)
+        {
+            string packageDir = null;
+            packageRoot = null;
+            var pkginfo = _packagePathResolver.GetPackageInfo(packageId,version);
+            if (pkginfo != null)
+            {
+                packageDir  = pkginfo.PathResolver.GetInstallPath(packageId, version);
+                packageRoot = pkginfo.PathResolver.GetVersionListPath("");  //TODO Check with nuget  if this is the correct way to get the package root
```

Comment:
If this isn't fixed by the time you check this in, it would be a good idea to put this info into a comment until we can take that new nuget change.

---
In reply to: [94865939](https://github.com/dotnet/sdk/pull/591#discussion_r94865939) [](ancestors = 94865939)